packages/mds-db/policies.ts

```diff
@@ -218,7 +218,9 @@ export async function publishPolicy(policy_id: UUID) {
       throw new NotFoundError('cannot publish nonexistent policy')
     }
 
-    const publish_date = now()
+    if (!policy.start_date || policy.start_date < publish_date) {
+      throw new ConflictError('publish_date must be before start_date')
```

Comment:
Could probably make this error message a bit more human-helpful by changing it to something like 'Policies cannot be published after their scheduled start_date.'